jupyter_server/services/kernels/kernelmanager.py

```diff
@@ -303,13 +300,12 @@ def shutdown_kernel(self, kernel_id, now=False):
             type=self._kernels[kernel_id].kernel_name
         ).dec()
 
-        return super(MappingKernelManager, self).shutdown_kernel(kernel_id, now=now)
+        return run_sync(self.super.shutdown_kernel(self, kernel_id, now=now))
```

Comment:
I think this should be restored (or use the `self.super.method(self, ...)` form of things) and we add an async form of shutdown_kernel to `AsyncMappingKernelManager`. Once that is done, I think we might be good.

I'd rather we not penalize kernel shutdown for async users by blocking.